src/OpenLoco/src/Ui/Windows/Vehicle.cpp

```diff
@@ -148,28 +153,57 @@ namespace OpenLoco::Ui::Windows::Vehicle
 
     namespace Details
     {
-        static constexpr Ui::Size32 kMinWindowSize = { 192, 148 };
+        static constexpr Ui::Size32 kMinWindowSize = { 192, 166 };
```

Comment:
min window width needs to be increased so the text is not cut off by the paint buttons:
![image](https://github.com/user-attachments/assets/9840d38d-86a0-49c2-9de2-d55cafded93b)

Reply:
What would be the purpose? Vanilla also truncates.

![image](https://github.com/user-attachments/assets/81384f40-c129-4762-a062-27058e6c5df6)

Reply:
copy existing bad code/design/practice is also just a straight up bad practice. we should always fix/improve where necessary and this is an obvious place, considering you're already touching the window bounds

Reply:
What's the practical benefit to forcing the window larger? The window already cleanly handles being smaller than displaying all the text. It just means people who want the window that small can't do it any more.